proc/proc_test.go

```diff
@@ -32,8 +32,12 @@ func TestMain(m *testing.M) {
 }
 
 func withTestProcess(name string, t testing.TB, fn func(p *Process, fixture protest.Fixture)) {
+	withTestProcessLaunchArgs(name, t, ".", fn)
+}
+
+func withTestProcessLaunchArgs(name string, t testing.TB, wd string, fn func(p *Process, fixture protest.Fixture)) {
```

Comment:
I think it would be better to add a `wd string` argument to `withTestProcessArgs` instead of making another helper function.